pkg/feature/storage/v2/sql/flag_trigger/delete_flag_trigger.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE flag_triggers
+SET deleted   = 1,
```

Comment:
I forgot to mention.
We don't need the `deleted` flag.
When the user wants to delete it, we will delete it permanently. So please update the query.